src/Form/test-cases/newgrounds_signup.html

```diff
@@ -0,0 +1,58 @@
+<!-- https://www.newgrounds.com/passport/signup/new -->
+<form action="/passport/appsession/94425869.887beedb5cf42632719cfa011fb607e92448072f882d24/signup/1" method="post">
+    <input type="hidden" name="register" value="web">
+    <div>
+        <input class="PassportTextfield" placeholder="Select a username" type="text" name="new_username" value=""
+            maxlength="20" required="" data-manual-scoring="username">
```

Comment:
Should this be `username` or `unknown`? The signup form here has both a `username` and an `emailAddress` field; the login form accepts either.

Reply:
I'd say `username`.